index.html

```diff
@@ -758,7 +758,46 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 the cost of implementation complexity.
       </dd>
     </dl>
-
+    <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. For instance, many proof formats
+cannot reveal selected attribute values in a verifiable credential;
+they can only reveal all (or none).
+    </p>
+    <p>
+Zero-Knowledge Proofs (ZKPs) are a proof format that enables
+privacy-preserving and data-minimization features in
+verifiable presentations, such as selective disclosure and predicate proofs.
+    </p>
+    <p>
+Selective disclosure is the ability of a holder to reveal a subset of
+the attributes of a verifiable credential in a verifiable presentation.
+A presentation based on ZKPs only contains those attributes and
+associated values that are required to satisfy the presentation requirements. 
+    </p>
+    <p>
+ZKP predicate proofs are a type of proof associated with an attribute.
+Predicate proofs include comparisons, such as
+"greater than", "less than", "not equal", "range", 
+"set membership", and "set non-membership".
+Predicate proofs can be
+constructed without requiring the issuer to create special fields, such as
+"age-under-18", "age-over-21", "age-over-60", in the verifiable credential
+at the time the credential was issued.
+The holder can create a predicate proof based on "birth-date" in the
+verifiable presentation derived from the verifiable credential to
+cryptographically prove to the verifier that the holder's age is greater than
+(or less than) a specified number without revealing the holders "birth-date".
+    </p>
+    <p>
+Verifiable credentials based on ZKPs are also quantum-resistant after
+presentation.
+    </p>
+    <p>
+A drawback of ZKPs is that they are more complex and larger than some
```

Comment:
I am not a fan of `ZKPs` and generally prefer a phrase such as `zero-knowledge proof mechanisms` which better ensures that the language surrounding the phrase remains accurate.

Reply:
Made changes to reflect this suggestion.